### DISCUSSION TITLE
Stream to specific destination automatically when OBS is streaming.

### DISCUSSION THREAD

**Discussion by jjmmfi** [sorayuki/obs-multi-rtmp#80] [Ideas]
Like your plugin very much! Awesome work! I just got one idea about a new feature:

Would be great to add an option to start and stop streams automatically while OBS is streaming. 
For example, just add a checkbox to every destination where you can choose do you want to start and stop broadcasting to that destination automatically when OBS starts/stops streaming. 

Sorry if my English is not the best. I am a Finn.

And thank you very much!

Best regards,
Jeremias Manninen

**Comment by bStyler** [comment]
@jeremiasmanninen great idea.

I just tested the plugin on Windows with OBS 26.1.1 and I face the same challenge.

I am using the `--startstreaming` [launch parameter](https://obsproject.com/wiki/Launch-Parameters) in a Windows shortcut, and my stream is automatically streaming.

However, I can't set it to **automatically** start streaming the multiple outputs configured by the plugin. That could be very helpful.

@sorayuki is building that improvement difficult for you? 

If so, it would be great to also avoid this error 👇 
![image](https://user-images.githubusercontent.com/7829256/119486085-8ae97d80-bd60-11eb-9822-80980878d5a1.png)
So all instances will stream automatically.

Thank you!
